practice-one/.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
```

Comment:
What does this line mean?

Reply:
i just make a copy on editorconfig.org. Line 11 explain how to apply for multiple files